ui/src/services/k8s/api.js

```diff
@@ -1,11 +1,9 @@
 import ApiClient from '../ApiClient';
 
-import {
-  Config,
-  CoreV1Api,
-  CustomObjectsApi,
-  StorageV1Api
-} from '@kubernetes/client-node';
+import { Config } from '@kubernetes/client-node/dist/config';
```

Comment:
Shouldn't this be `@kubernetes/client-node/dist/browser/config`? Or will the replacements defined in said package's `package.json` also handle this case?

Reply:
> Could we, given this, only depend on `lodash.isempty` and `lodash.sortby` instead of all of `lodash`? Or is that an indirect dependency anyway?
> 
> If not, given the two security-related PRs we had recently due to bugs in `lodash`, limiting how much of `lodash` we depend on may be worth that effort.

Done :)

Reply:
> Shouldn't this be `@kubernetes/client-node/dist/browser/config`? Or will the replacements defined in said package's `package.json` also handle this case?

I just tested both, we can use this one. Actually, if we are not using this specific import, I don't understand why we forked `@kubernetes/client-node`. The project seems to work on both and we are not using `browser` stuff from the beginning I think (or we thought we are using it ?)
Any ideas @carlito-scality ?

Reply:
Exactly, so that's why the *non-browser-version* also worked from the start, most likely.

It *is* a kludge. I'd love to, at some point, work with upstream to make their library 'cross-platform' out of the box, both NodeJS as well as browser.